sequence2_modified.gb

```diff
@@ -0,0 +1,161 @@
+LOCUS       AF165912                5485 bp    DNA     linear   PLN 29-JUL-1999
```

Comment:
This looks like an accidental check-in duplicate of the test file.

Reply:
Yep, removed.